common/src/lru_cache.rs

```diff
@@ -92,7 +92,7 @@ where
             None =>
             /* unexpected */
             {
-                return None
+                return None;
```

Comment:
Was this fixed manually? If not, why didn't it get caught in Jenkins?

Reply:
Newly  written file made this change when testing. I see there's a possibility of code improvement here.

Instead of matching over result and re-creating Option of result, we can directly return result by changing it to if condition block where if something is found then make changes to ordered_keys.